src/sections/hero.liquid

```diff
@@ -0,0 +1,62 @@
+{% assign settings = section.settings %}
+{% assign img_url = settings.background_image | img_url: 'master' %}
+
+<section class="section">
+  <div class="hero flex-column center-both full-height" style="{% if settings.background_image %} background-image: url({{ img_url}}; {% endif %}">
```

Comment:
not sure this necessarily has to be full height, but fine for now—can change it later. depending on the client's content we might want to allow a separate mobile image upload.